src/lib/StorePrototype.js

```diff
@@ -15,80 +15,64 @@ import AppDispatcher from './fu-dispatcher'
 
 function factory(registerAction, extendType, registerMessageAction) {
   var StorePrototype = assign(EventEmitter.prototype, {
```

Comment:
This file was a mess. I cleaned it up and organized the methods so it looks better to outsiders